packages/microsite/src/cli/microsite-build.ts

```diff
@@ -2,7 +2,8 @@ import { build as buildProject } from "snowpack";
 import { dirname, resolve } from "path";
 import glob from "globby";
 import arg from "arg";
-import { GetModuleInfo, rollup } from "rollup";
+import { GetModuleInfo, rollup, RenderedChunk, Plugin } from "rollup";
+import { visit } from "recast";
```

Comment:
fyi: I picked this library because it can do both things: traverse and write the tree. In case you care about the bundle size there are more lightweight alternatives just for traversing (e.g. [estree-walker](https://github.com/Rich-Harris/estree-walker))